cyber/mainboard/module_argument.cc

```diff
@@ -62,7 +62,6 @@ void ModuleArgument::ParseArgument(const int argc, char* const argv[]) {
 }
 
 void ModuleArgument::GetOptions(const int argc, char* const argv[]) {
```

Comment:
opterr is an external variable. Better not to remove it.

Reply:
opterr is designed upon getopt(3) rather than getopt_long(3).